crates/oxc_linter/src/rules/jest/consistent_test_it.rs

```diff
@@ -16,66 +18,51 @@ use crate::{
     },
 };
 
-fn consistent_method(x1: &str, x2: &str, span: Span) -> OxcDiagnostic {
+fn consistent_method(preferred_method: &str, other_method: &str, span: Span) -> OxcDiagnostic {
     OxcDiagnostic::warn("Enforce `test` and `it` usage conventions")
-        .with_help(format!("Prefer using {x1:?} instead of {x2:?}"))
+        .with_help(format!("Prefer using {preferred_method:?} instead of {other_method:?}"))
         .with_label(span)
 }
 
-fn consistent_method_within_describe(x1: &str, x2: &str, span: Span) -> OxcDiagnostic {
+fn consistent_method_within_describe(
+    preferred_method: &str,
+    other_method: &str,
+    span: Span,
+) -> OxcDiagnostic {
     OxcDiagnostic::warn("Enforce `test` and `it` usage conventions")
-        .with_help(format!("Prefer using {x1:?} instead of {x2:?} within describe"))
+        .with_help(format!(
+            "Prefer using {preferred_method:?} instead of {other_method:?} within describe"
+        ))
         .with_label(span)
 }
 
-#[derive(Debug, Clone, Copy, Eq, PartialEq)]
+#[derive(Debug, Clone, Copy, Eq, PartialEq, Deserialize, JsonSchema, Serialize)]
+#[serde(rename_all = "lowercase")]
 enum TestCaseName {
-    Fit,
     IT,
     Test,
-    Xit,
-    Xtest,
```

Comment:
was moving these different test methods deliberate? if so why?

Reply:
i see, these options were actually supported. but only by us, not the original rule. lets remove the support